src/Accounts/Accounts/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 ## Upcoming Release
 * Added new parameter `-ClaimsChallenge` to `Connect-AzAccount` to support claims challenge authentication for MFA.
 * Refined the error message when a cmdlet fails because of policy violations about Multi-Factor Authentication (MFA) to provide more actionable guidance.
+* Added Server-Side Pagination Support for `Invoke-AzRestMethod` command via `-FollowNextLink` parameter.
```

Comment:
Github sometimes incorrectly merge conflicts silently for Changelog.md, a good practice to avoid that is to always put new added release notes right under "## Upcoming Release" (line 22), thus conflicts will be guaranteed detected